dataactbroker/handlers/fileHandler.py

```diff
@@ -141,18 +141,20 @@ def get_signed_url_for_submission_file(self):
         """ Gets the signed URL for the specified file """
         try:
             self.s3manager = s3UrlHandler()
-            file_name = request.args.get('file') + ".csv"
-            submission_id = request.args.get('submission')
+            safe_dictionary = RequestDictionary(self.request)
+            file_name = safe_dictionary.getValue("file") + ".csv"
+            submission_id = safe_dictionary.getValue("submission")
             submission = self.jobManager.getSubmissionById(submission_id)
             # Check that user has access to submission
             # If they don't, throw an exception
             self.check_submission_permission(submission)
 
+            response_dict = {}
             if self.isLocal:
-                return send_from_directory(self.serverPath, file_name)
+                response_dict["url"] = os.path.join(self.serverPath, file_name)
```

Comment:
Should this _always_ join with `/` instead? If the server's running windows, we'd still want forward slashes in the generated url, right?